examples/fedpca_examples/perform_pca/config.yaml

```diff
@@ -0,0 +1,12 @@
+# Parameters that describe server
+n_server_rounds: 1 # The number of rounds to run FL
+
+# Parameters that describe clients
+n_clients: 4 # The number of clients in the FL experiment
+batch_size: 32 # The batch size for client training
```

Comment:
I'd change the comment on this variable here, since we're not really using the batch_size for training in the PCA example.